docs/core/tools/dotnet-build.md

```diff
@@ -45,11 +45,11 @@ compilation process to be incremental:
 - not load compilation tools from PATH (for example, resgen, compilers)
 - use only known compilers (csc, vbc, fsc)
 
-In order to build an executable application, you need a special configuration section in your project.json file:
+In order to build an executable application instead of a DLL, you need a [special configuration](project-json.md#emitentrypoint) section in your project.json file:
```

Comment:
@mairaw yes and no. I think there are two questions here:

1. How do I produce a runnable artifact, e.g. an application?
2. How do I produce an "exe" file, that is, a native executable that I can "just run" on a given operating system?

For #1 above, it is true, you need the `emitEntryPoint` setting, but, as @svick mentions, you would still get a DLL.

For #2, the only way is to create a self-contained deployment. 

Hope this makes sense. In general, I would go with @svick's suggestion and just say "library" instead of a "DLL".